skietbaan_fe/src/components/List.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from "react";
+import { Table } from "react-bootstrap";
+import '../components/ListStyle.css'
+class List extends Component {
+  constructor(props){
+    super(props);
+    this.state = {
+      posts :[]
+    }
+  }
+  componentWillMount() {
+    fetch('http://skietbaan.retrotest.co.za/api/User')
+    .then(res => res.json())
+    .then(data =>this.setState({posts : data}));
+  }
+  render() {
+    const postItems = (
+      <Table striped hover condensed>
+        <thead>
+         <tr>
+          <td><b>USER_ID</b></td>
+          <td><b>USERNAME</b></td>
+          <td><b>EMAIL</b></td>
+          </tr>
+        </thead>
+        <tbody>
+          {this.state.posts.map(post => (
+            <tr key={post.id}>
+            <td><h5>{post.id}</h5></td>
+              <td><h5>{post.username}</h5></td>
+              <td>{post.email}</td>
```

Comment:
why are you listing users emails and usernames here? Does this no violate user privacy. Also why does server even return that. Please also ensure that, the user object that is returned does not contain the password hash as well

Reply:
this was for testing if i retrieve the data  it will change once we have real data in database

Reply:
If its for testing why is it being committed?

Reply:
i have changed it